docs/checks/checks.yaml

```diff
@@ -32,8 +33,9 @@ checks:
         your project activity and maintenance commitment.
   Automatic-Dependency-Update:
     risk: High
+    short: Determines if a project has dependencies automatically updated.
```

Comment:
maybe
project has enabled automatic dependency updates (both in short and full description)

Reply:
we could also change this check to be `dependency update tool` or more generally `dependency management tool`. The updates are not automatic and this seems to confuse some users, because the check's name seems to go against dependency pinning.
Can so that later... but curious what you think

Reply:
yes "project uses a dependency update tool" is better.

Reply:
shall I update the check's name as well -> `Dependency-Update-Tool`?